ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -87,59 +119,73 @@ TFCore::getModelPath ()
   return model_path;
 }
 
+/**
+ * @brief	the definition of a deallocator method
+ */
+static void
+DeallocateBuffer (void* data, size_t t) {
+  std::free (data);
+}
+
 /**
  * @brief	load the tf model
  * @note	the model will be loaded
  * @return 0 if OK. non-zero if error.
- *        -1 if the modelfile is not valid(or not exist).
- *        -2 if the pb file is not loaded.
- *        -3 if the input properties are different with model.
- *        -4 if the Tensorflow session is not initialized.
- *        -5 if the Tensorflow session is not created.
+ *        -1 if the pb file is not loaded.
+ *        -2 if importing graph is failed.
+ *        -3 if the Tensorflow session is not created.
  */
 int
 TFCore::loadModel ()
 {
 #if (DBG)
   gint64 start_time = g_get_real_time ();
 #endif
+  gsize file_size;
+  gchar *content = nullptr;
+  GError *file_error = nullptr;
 
-  Status status;
-  GraphDef graph_def;
+  g_assert (model_path != nullptr);
 
-  if (!g_file_test (model_path, G_FILE_TEST_IS_REGULAR)) {
-    g_critical ("the file of model_path (%s) is not valid (not regular)\n", model_path);
+  g_file_get_contents (model_path, &content, &file_size, &file_error);
```

Comment:
plz check return result of g_file_get_contents(), not the error. 
(you may set NULL if you dont want to get the error message)

https://developer.gnome.org/glib/stable/glib-File-Utilities.html#g-file-get-contents
TRUE on success, FALSE if an error occurred

Reply:
it's applied